openshift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This will produce and image to be used in OpenShift
+# Build should be triggered from the repo root like:
+# docker build -f Dockerfile --tag 172.30.1.1:5000/myproject/resultsdb-updater:latest --build-arg resultsdb_updater_rpm=resultsdb-updater-2.4.0-1.el7eng.noarch.rpm .
```

Comment:
This would require the repo to be made available during build time. To keep things flexible that would probably also mean an build time argument.

Btw, this tries to align with the work done by @danc86 on [waiverdb](https://pagure.io/waiverdb), where iirc a Dockerfile and template similar to these was used in waiverdb CI.